src/cloud-api-adaptor/podvm/Makefile

```diff
@@ -83,6 +83,9 @@ $(IMAGE_FILE): $(BINARIES) $(FILES) setopts
 	if [ "${SE_BOOT}" = "1" ] && [ "${ARCH}" = "s390x" ]; then \
 		qemu-img create -f qcow2 "se-${IMAGE_FILE}" 100G; \
 	fi
+	if [ "${ARCH}" = "x86_64" ]; then \
+		packer plugins install github.com/hashicorp/qemu; \
```

Comment:
nit, any idea if it's possible to pin the version of the plugin used here? if yes, if would be good to pin it to avoid sudden bugs or licence change

Reply:
I have modified code accordingly, could you please take a look and suggest. Thank you